swri_transform_util/src/nodes/dynamic_transform_publisher.cpp

```diff
@@ -36,7 +36,7 @@
 #include <rcl_interfaces/msg/floating_point_range.hpp>
 #include <rcl_interfaces/msg/parameter_descriptor.hpp>
 #include <tf2_ros/transform_broadcaster.h>
-#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
```

Comment:
This, and all other files that include `tf2_geometry_msgs.hpp`, will fail to compile on ROS Galactic and earlier because there the file is named `tf2_geometry_msgs.h`.  Unfortunately, while that file exists in Humble, including it will print a warning that cannot be suppressed, which is incredibly annoying.

Probably the best way to make this work across different ROS2 releases is to modify the CMakeLists.txt to pull the `ROS_DISTRO` environment variable and set that in a compiler definition, then use preprocessor macros here to conditionally include either `tf2_geometry_msgs.h` or `tf2_geometry_msgs.hpp`.